indico/htdocs/sass/partials/_timelines.scss

```diff
@@ -166,3 +166,61 @@
         margin-top: 0;
     }
 }
+
+ul.deadline-timeline {
```

Comment:
This has nothing to do with the timeline component. We need to find a different file to place this and, probably, a better naming not to confuse it with the timeline component.